pkgs/top-level/all-packages.nix

```diff
@@ -31695,4 +31695,6 @@ in
   xcolor = callPackage ../tools/graphics/xcolor { };
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  bsp-layout = callPackage ../tools/misc/bsp-layout {};
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
Added before `buildtorrent` in c844ab7.